ganttproject/src/net/sourceforge/ganttproject/export/ExporterToExcel.java

```diff
@@ -0,0 +1,54 @@
+package net.sourceforge.ganttproject.export;
+
+import biz.ganttproject.core.option.GPOptionGroup;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+
+import java.awt.*;
+import java.io.File;
+import java.util.List;
+
+/**
+ * Created by valentin on 15.03.17.
```

Comment:
We usually write `@author <email> (<your real name>)` or vice versa, `@author <Your real name> (<email>)`